common/tools/dev-tool/src/commands/samples/prep.ts

```diff
@@ -68,11 +68,6 @@ async function enableLocalRun(
 
     let relativePath = new Array(depth).fill("..").join("/");
 
-    if (isTs) {
-      // TypeScript imports should use src directly
-      relativePath += "/src";
-    }
-
```

Comment:
This was done before for reasons that @witemple-msft does not remember and it is causing the storage-file-datalake samples to fail compiling because storage-common is just a simple folder of source files instead of being a package with a `package.json` (unlike keyvault-common) so compiled files there do not have access to tslib.